docs/CONTRIBUTING.md

```diff
@@ -376,6 +391,14 @@ Then start the background worker (which will process uploaded READMEs):
 cargo run --bin background-worker
 ```
 
+Since these programs use the logging framework, you can enable debug logging by
+setting the `RUST_LOG` environment variable to `debug` before running them, for
+example:
+
+```console
+RUST_LOG=debug cargo run --bin background-worker
+```
+
```

Comment:
```suggestion
Since crates.io is using the `tracing` crate, you can enable debug logging by
setting the `RUST_LOG` environment variable to `debug` before running them, for
example:

```console
RUST_LOG=debug cargo run --bin background-worker
```

```